src/main/java/org/gridsuite/filter/server/FilterController.java

```diff
@@ -203,4 +203,16 @@ public ResponseEntity<List<IdentifiableAttributes>> evaluateFilter(@RequestParam
         List<IdentifiableAttributes> identifiableAttributes = service.evaluateFilter(filter, networkUuid, variantId);
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(identifiableAttributes);
     }
+
+    @GetMapping(value = "/filters/{filterUuid}/evaluate", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Why adding this new endpoint ?

The existing endpoint "/filters/{id}/export" do the same job ...

Reply:
it's about the parameter, in the other endpoint you need to provide the filter which is not the case in the new endpoint, so if you don't have the filter but only its id you can still evaluate it.

Reply:
No, in the endpoint "/filters/{id}/export", we provide a filter id ...